core/src/main/scala/io/udash/properties/single/ReadableProperty.scala

```diff
@@ -65,6 +74,16 @@ trait ReadableProperty[+A] {
     new CombinedProperty[A, B, O](this, property, combiner)
 }
 
+final class MirrorProperty[B: PropertyCreator](origin: ReadableProperty[B]) {
```

Comment:
package private maybe?

Reply:
What would the return type of `def mirror` be then? More importantly, I think it would be hard to make the implicits visible by default then.